content/lighthouse/configure-lighthouse-reports.md

```diff
@@ -0,0 +1,110 @@
+---
+title: "Configure Lighthouse Audits"
```

Comment:
The problem is that in the Lighthouse documentation, these checks are referred to as audits. I struggled with being consistent with our taxonomy, and consistent with the tool's taxonomy. There are also keyword variants with both that we'd want to capture for search purposes.

I thought a good compromise was to refer to the act of completing these checks as audits, since that is consistent with Google's terminology, and the output as reports - i.e. the documents generated as a result of the checks are the reports. I think that tracks with our UI using "Report" because that's all you're seeing in the UI - the results of the audit. But configuring them is setting up the audit, because you're establishing what things to check to generate the output. I haven't looked at the API calls available for this; is that division of terms a problem there?

Reply:
Our API docs don't touch on the individual audits, other than to briefly mention and link off to external documentation for how to specify your own custom lighthouse configuration. Other than that, we really only deal with the aggregated results. Even the scores shown in the database are an aggregation of all of the audits for that section.

So, I think it's accurate to say that for our purposes, we can stick with "Lighthouse Report[s]" and still be consistent with the upstream vocabulary.